docs/reference/packages.rst

```diff
@@ -14,12 +14,12 @@ packages:
 
 1. Python modules and packages are turned into distribution package
    archives, and published at the Python Package Index (PyPI).
-2. `upip` package manager can be used to install a distribution package
-   on a `MicroPython port` with networking capabilities (for example,
-   on the Unix port).
+2. `upip` package manager is used to install a distribution package
+   on a `MicroPython port` with networking capabilities. You can run upip
+   on your microcontroller or your Unix port.
 3. For ports without networking capabilities, an "installation image"
-   can be prepared on the Unix port, and transferred to a device by
-   suitable means.
+   can be prepared on the Unix port. You can also install packages to your development
+   environment (a Unic\MacOs\Winows Port) then transfer the code to your device. see: `Cross-installing packages`_
```

Comment:
`Unic` -> `Unix`.

But the text became less clear now imo: it used to say 'you can prepare images and then transfer'. But now it says 'you can prepare an images on the unix port' and 'also you can install packages on the unix\osx\windows port then transfer' so that 'also' refers to something which isn't there anymore because you deleted the original text about transferring and it's unclear why both sentences don't mention the same ports. Just change the original sentence a bit?

Reply:
See above... I think the new edit is succinct and clear.